tests-perf/ops/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.9-alpine3.13
+
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apk add --no-cache bash build-base git libtool cmake autoconf automake gcc musl-dev postgresql-dev g++ libexecinfo-dev make libffi-dev libmagic libcurl curl-dev rust cargo && rm -rf /var/cache/apk/*
+
+# update pip
+RUN python -m pip install wheel
+RUN python -m pip install --upgrade pip
+
+RUN set -ex && mkdir /app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN set -ex && pip3 install -r requirements_for_test.txt
```

Comment:
If you aren't changing the `requirements_for_test.txt` that often you can speed up build times for this Dockerfile by switching up how you copy files into this container, and taking advantage of layer caching. 

The following will only run pip3 install if the requirements file changes not if any of your python files change which it does now. 

```
COPY requirements_for_test.txt ./

RUN set -ex && pip3 install -r requirements_for_test.txt

COPY . ./
```

You can use the `./` instead of `/app` because WORKDIR changes the current directory to `/app`

https://docs.docker.com/engine/reference/builder/#workdir

Please note this is not a blocker in any way just a possible improvement

Reply:
Sure this observation is noted abs will be addressed